### ISSUE TITLE
Merging Grids Crashing Servers

### ISSUE THREAD

**Issue by subbob** [DraconisCluster/DX-Issues#6]
** tl; dr **
As of last week, merging grids often results in crashing an entire instance

** Full info **
I had a broken ship and attempted to merge it. I always convert the target ship to station first, prior to merge. I've done this (merged ships) well over a dozen times since late November with no issues.  Last week, as soon as the merge blocks engaged, the client locked up and instance crashed. After it was restored, I tried again, and recorded it this time.  Immediately crashed the instance. The video is in the ticket referenced below.

It happened again two days ago (different ship). My gun platform was blown off my ship in DX-7. I tried merging it to my ship to save it from cleanup.  Client froze and instance crashed as soon as merge blocks touched.  A related ticket was opened by another player to restore a grid lost due to the server crash. I believe it was the 2nd ticket referenced below.

**Screenshots**
N/A

**References**
(1) [ticket 48992 by @subbob](https://discord.com/channels/516135382191177728/697798386962661457/1190884567280779394) 
(2) [ticket 49085 by @hobobot](https://discord.com/channels/516135382191177728/697798386962661457/1192345159174733825)

**Comment by Terif1** [comment]
I have a hunch on this actually, unless much mistaken these were silent crashes but I did find a silent crash cause on die which would fit this particular bill. Waiting for the dev to update his plugin with the fix, once that's done we will need to see if this fixes this due to not being able to debug the expanse servers properly

**Comment by Terif1** [comment]
I've also disabled the thing that causes this to happen for the moment, hopefully

**Comment by subbob** [comment]
[Video of 2nd crash during merge attempt on 26 December](https://cdn.discordapp.com/attachments/1021881558879191111/1192858421272252416/MergeCrashDX7.mp4?ex=65aa9af9&is=659825f9&hm=89250048205e75afdc918366bc3695f899faf29f56f0482f2ed840be2e924866&)

Sounds like you're already narrowing in on the cause. Providing this for the record.

**Comment by draconb** [comment]
I had a grid that would constantly crash when merging months ago in the old inner dx7 instance. I just left it with connectors as it was a resupply fob so didn't need to be merged. I can try again with the grid if we need a test, not sure if its still crashing since I haven't used that grid in a few months now since the server map change.

**Comment by kontu** [comment]
> I had a grid that would constantly crash when merging months ago in the old inner dx7 instance. I just left it with connectors as it was a resupply fob so didn't need to be merged. I can try again with the grid if we need a test, not sure if its still crashing since I haven't used that grid in a few months now since the server map change.

try it!

**Comment by Terif1** [comment]
if anyone has a moment, is this still an issue? the thing I thought was the issue should have been fixed

**Comment by subbob** [comment]
In the last week I have done several merges without issue. I also have not seen anything in game chat about people crashing the server doing a merge.

**Comment by Terif1** [comment]
gonna say this is fixed with the torchmonitor fix
